pymc/model/core.py

```diff
@@ -1243,16 +1170,16 @@ def set_data(
 
     def register_rv(
         self,
-        rv_var,
-        name,
+        rv_var: TensorVariable,
```

Comment:
Technically, RVs only need to be Variable, they need not be TensorVariable
```suggestion
        rv_var: Variable,
```